src/LightBulbAccessory.ts

```diff
@@ -71,30 +74,38 @@ export default class LightBulbAccessory {
       .onGet(Brightness.get.bind(this))
       .onSet(Brightness.set.bind(this));
 
-    this.service
-      .getCharacteristic(this.platform.Characteristic.ColorTemperature)
-      .setProps({
-        minValue: HOME_KIT_VALUES.min,
-        maxValue: HOME_KIT_VALUES.max
-      })
-      .onGet(ColorTemperature.get.bind(this))
-      .onSet(ColorTemperature.set.bind(this));
+    if (supportsColorTemp == true) {
```

Comment:
Don't need to include the `== true` you can only do `if(supportsColorTemp)`. Also, in JS/TS, never use `==`, use `===`, because `'1' == 1 -> true` and `'1' === 1 -> false`